homeassistant/components/unifi/switch.py

```diff
@@ -448,3 +475,58 @@ def device_info(self) -> DeviceInfo:
             model="UniFi Network",
             name="UniFi Network",
         )
+
+
+class UniFiOutletSwitch(UniFiBase, SwitchEntity):
+    """Representation of a outlet relay."""
+
+    DOMAIN = DOMAIN
+    TYPE = OUTLET_SWITCH
+
+    _attr_device_class = SwitchDeviceClass.OUTLET
+
+    def __init__(self, device, controller, index):
+        """Set up outlet switch."""
+        super().__init__(device, controller)
+
+        self._outlet_index = index
+
+        self._attr_name = f"{device.name or device.model} {device.outlets[index].name}"
+        self._attr_unique_id = f"{device.mac}-outlet-{index}"
```

Comment:
I'm assuming the mac is the mac of the outlet device and the index is the outlet number so its stable and won't every change since its physically representing the device.

Reply:
The Mac is definitely stable, and I assume it would be useless design if the index could change